### PR TITLE
playground: add links to playground

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Add links to TiDB Playground.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ aliases: ['/tidbcloud/beta/tidb-cloud-quickstart']
 
 This tutorial guides you through an easy way to get started with your TiDB Cloud.
 
+In addition, you can have a try of TiDB on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-quickstart.md:14]:
```suggestion
In addition, you can have a try of TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ This guide walks you through the quickest way to get started with TiDB. For non-
 - [Deploy a local test cluster](#deploy-a-local-test-cluster) (for macOS and Linux)
 - [Simulate production deployment on a single machine](#simulate-production-deployment-on-a-single-machine) (for Linux only)
 
+In addition, you can have a try of TiDB on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (hfxsd) [inline quick-start-with-tidb.md:14]:
```suggestion
In addition, you can have a try of TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```

File: mysql-compatibility.md [hunk 3]
```diff
@@ -35,6 +35,8 @@ In addition, TiDB does not support the MySQL replication protocol, but provides
 
 </CustomContent>
 
+You can have a try of TiDB on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (hfxsd) [inline mysql-compatibility.md:38]:
```suggestion
You can have a try of TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```

File: basic-features.md [hunk 4]
```diff
@@ -8,6 +8,8 @@ aliases: ['/docs/dev/basic-features/','/tidb/dev/experimental-features-4.0/']
 
 This document lists the features supported in different TiDB versions, including [Long-Term Support (LTS)](/releases/versioning.md#long-term-support-releases) versions and [Development Milestone Release (DMR)](/releases/versioning.md#development-milestone-releases) versions after the latest LTS version.
 
+You can have a try of TiDB on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (hfxsd) [inline basic-features.md:11]:
```suggestion
You can have a try of TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```

File: _docHome.md [hunk 5]
```diff
@@ -103,6 +103,12 @@ Documentation for TiDB Cloud application developers
 
 <DocHomeCardContainer>
 
+<DocHomeCard href="http://play.tidbcloud.com" icon="playground" label="TiDB Playground">
+
+One-stop and interactive experience of TiDB's capabilities WITHOUT registration
```
Comment (ChenlingLu) [inline _docHome.md:108]: I like the 'WITHOUT', it makes this sentence more powerful!

**Review by ChenlingLu — COMMENTED** [review]
LGTM

File: mysql-compatibility.md [hunk 6]
```diff
@@ -35,6 +35,8 @@ In addition, TiDB does not support the MySQL replication protocol, but provides
 
 </CustomContent>
 
+You can have a try of TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (lilin90) [inline mysql-compatibility.md:38]:
```suggestion
You can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```

File: quick-start-with-tidb.md [hunk 7]
```diff
@@ -11,6 +11,8 @@ This guide walks you through the quickest way to get started with TiDB. For non-
 - [Deploy a local test cluster](#deploy-a-local-test-cluster) (for macOS and Linux)
 - [Simulate production deployment on a single machine](#simulate-production-deployment-on-a-single-machine) (for Linux only)
 
+In addition, you can have a try of TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (lilin90) [inline quick-start-with-tidb.md:14]:
```suggestion
In addition, you can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 8]
```diff
@@ -11,6 +11,8 @@ aliases: ['/tidbcloud/beta/tidb-cloud-quickstart']
 
 This tutorial guides you through an easy way to get started with your TiDB Cloud.
 
+In addition, you can have a try of TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-quickstart.md:14]:
```suggestion
In addition, you can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```

File: basic-features.md [hunk 9]
```diff
@@ -8,6 +8,8 @@ aliases: ['/docs/dev/basic-features/','/tidb/dev/experimental-features-4.0/']
 
 This document lists the features supported in different TiDB versions, including [Long-Term Support (LTS)](/releases/versioning.md#long-term-support-releases) versions and [Development Milestone Release (DMR)](/releases/versioning.md#development-milestone-releases) versions after the latest LTS version.
 
+You can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (ChenlingLu) [inline basic-features.md]: http://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_features

File: mysql-compatibility.md [hunk 10]
```diff
@@ -35,6 +35,8 @@ In addition, TiDB does not support the MySQL replication protocol, but provides
 
 </CustomContent>
 
+You can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (ChenlingLu) [inline mysql-compatibility.md]: https://play.tidbcloud.com/?utm_source=docs&utm_medium=mysql_compatibility

File: quick-start-with-tidb.md [hunk 11]
```diff
@@ -11,6 +11,8 @@ This guide walks you through the quickest way to get started with TiDB. For non-
 - [Deploy a local test cluster](#deploy-a-local-test-cluster) (for macOS and Linux)
 - [Simulate production deployment on a single machine](#simulate-production-deployment-on-a-single-machine) (for Linux only)
 
+In addition, you can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (ChenlingLu) [inline quick-start-with-tidb.md]: https://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_quick_start

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 12]
```diff
@@ -11,6 +11,8 @@ aliases: ['/tidbcloud/beta/tidb-cloud-quickstart']
 
 This tutorial guides you through an easy way to get started with your TiDB Cloud.
 
+In addition, you can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/).
```
Comment (ChenlingLu) [inline tidb-cloud/tidb-cloud-quickstart.md]: https://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_cloud_quick_start

File: _docHome.md [hunk 13]
```diff
@@ -103,6 +103,12 @@ Documentation for TiDB Cloud application developers
 
 <DocHomeCardContainer>
 
+<DocHomeCard href="http://play.tidbcloud.com" icon="playground" label="TiDB Playground">
```
Comment (hfxsd) [inline _docHome.md:106]:
```suggestion
<DocHomeCard href="http://play.tidbcloud.com/?utm_source=xxxx&utm_medium=" icon="playground" label="TiDB Playground">
```

File: basic-features.md (hunk 9 above)
Comment (hfxsd) [inline basic-features.md:11]:
```suggestion
You can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_features).
```

File: mysql-compatibility.md (hunk 10 above)
Comment (hfxsd) [inline mysql-compatibility.md:38]:
```suggestion
You can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/?utm_source=docs&utm_medium=mysql_compatibility).
```

File: quick-start-with-tidb.md (hunk 11 above)
Comment (hfxsd) [inline quick-start-with-tidb.md:14]:
```suggestion
In addition, you can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_quick_start).
```

File: tidb-cloud/tidb-cloud-quickstart.md (hunk 12 above)
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-quickstart.md:14]:
```suggestion
In addition, you can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_cloud_quick_start).
```

File: _docHome.md [hunk 14]
```diff
@@ -103,6 +103,12 @@ Documentation for TiDB Cloud application developers
 
 <DocHomeCardContainer>
 
+<DocHomeCard href="http://play.tidbcloud.com/?utm_source=docs&utm_medium=home_more_resource" icon="playground" label="TiDB Playground">
```
Comment (hfxsd) [inline _docHome.md:106]:
```suggestion
<DocHomeCard href="https://play.tidbcloud.com/?utm_source=docs&utm_medium=home_more_resource" icon="playground" label="TiDB Playground">
```

File: basic-features.md [hunk 15]
```diff
@@ -8,6 +8,8 @@ aliases: ['/docs/dev/basic-features/','/tidb/dev/experimental-features-4.0/']
 
 This document lists the features supported in different TiDB versions, including [Long-Term Support (LTS)](/releases/versioning.md#long-term-support-releases) versions and [Development Milestone Release (DMR)](/releases/versioning.md#development-milestone-releases) versions after the latest LTS version.
 
+You can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_features).
```
Comment (hfxsd) [inline basic-features.md:11]:
```suggestion
You can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_features).
```

File: mysql-compatibility.md [hunk 16]
```diff
@@ -35,6 +35,8 @@ In addition, TiDB does not support the MySQL replication protocol, but provides
 
 </CustomContent>
 
+You can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/?utm_source=docs&utm_medium=mysql_compatibility).
```
Comment (hfxsd) [inline mysql-compatibility.md:38]:
```suggestion
You can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?utm_source=docs&utm_medium=mysql_compatibility).
```

File: quick-start-with-tidb.md [hunk 17]
```diff
@@ -11,6 +11,8 @@ This guide walks you through the quickest way to get started with TiDB. For non-
 - [Deploy a local test cluster](#deploy-a-local-test-cluster) (for macOS and Linux)
 - [Simulate production deployment on a single machine](#simulate-production-deployment-on-a-single-machine) (for Linux only)
 
+In addition, you can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_quick_start).
```
Comment (hfxsd) [inline quick-start-with-tidb.md:14]:
```suggestion
In addition, you can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_quick_start).
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 18]
```diff
@@ -11,6 +11,8 @@ aliases: ['/tidbcloud/beta/tidb-cloud-quickstart']
 
 This tutorial guides you through an easy way to get started with your TiDB Cloud.
 
+In addition, you can try out TiDB features on [TiDB Playground](http://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_cloud_quick_start).
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-quickstart.md:14]:
```suggestion
In addition, you can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_cloud_quick_start).
```

**Review by ChenlingLu — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
@hfxsd @lilin90 I have updated the https://tour.tidb.io to the new TiDB Playground in the latest commit https://github.com/pingcap/docs/pull/13744/commits/cb86f15d197d317223447982e03f9d97f19f2108. PTAL

File: basic-sql-operations.md [hunk 19]
```diff
@@ -10,7 +10,7 @@ TiDB is compatible with MySQL, you can use MySQL statements directly in most of
 
 <CustomContent platform="tidb">
 
-To experiment with SQL and test out TiDB compatibility with MySQL queries, you can [run TiDB directly in your web browser without installing it](https://tour.tidb.io/). You can also first deploy a TiDB cluster and then run SQL statements in it.
+To experiment with SQL and test out TiDB compatibility with MySQL queries, you can try [TiDB Playground](https://play.tidbcloud.com/?utm_source=docs&utm_medium=mysql_compatibility). You can also first deploy a TiDB cluster and then run SQL statements in it.
```
Comment (hfxsd) [inline basic-sql-operations.md:13]:
```suggestion
To experiment with SQL and test out TiDB compatibility with MySQL queries, you can try [TiDB Playground](https://play.tidbcloud.com/?utm_source=docs&utm_medium=basic-sql-operations). You can also first deploy a TiDB cluster and then run SQL statements in it.
```

File: develop/dev-guide-tidb-crud-sql.md [hunk 20]
```diff
@@ -19,7 +19,7 @@ Please make sure you are connected to a TiDB cluster. If not, refer to [Build a
 
 TiDB is compatible with MySQL, you can use MySQL statements directly in most cases. For unsupported features, see [Compatibility with MySQL](/mysql-compatibility.md#unsupported-features).
 
-To experiment with SQL and test out TiDB compatibility with MySQL queries, you can [run TiDB directly in your web browser without installing it](https://tour.tidb.io/). You can also first deploy a TiDB cluster and then run SQL statements in it.
+To experiment with SQL and test out TiDB compatibility with MySQL queries, you can try [TiDB Playground](https://play.tidbcloud.com/?utm_source=docs&utm_medium=mysql_compatibility). You can also first deploy a TiDB cluster and then run SQL statements in it.
```
Comment (hfxsd) [inline develop/dev-guide-tidb-crud-sql.md:22]:
```suggestion
To experiment with SQL and test out TiDB compatibility with MySQL queries, you can try [TiDB Playground](https://play.tidbcloud.com/?utm_source=docs&utm_medium=basic-sql-operations). You can also first deploy a TiDB cluster and then run SQL statements in it.
```

File: _docHome.md [hunk 21]
```diff
@@ -103,6 +103,12 @@ Documentation for TiDB Cloud application developers
 
 <DocHomeCardContainer>
 
+<DocHomeCard href="https://play.tidbcloud.com/?utm_source=docs&utm_medium=home_more_resource" icon="playground" label="TiDB Playground">
```
Comment (lilin90) [inline _docHome.md:106]:
```suggestion
<DocHomeCard href="https://play.tidbcloud.com/?utm_source=docs&utm_medium=home_more_resources" icon="playground" label="TiDB Playground">
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**